### PR TITLE
prevent click and drag interactions from triggering through overlays

### LINKED ISSUE #2452: On iPad, click and drag interactions triggered through overlays
See Slate ticket 2391

### PR DESCRIPTION
**Pull request by ztsai**
closes #2452
demo: http://jsfiddle.net/ztsai/gzLqekvh/5/

### CODE REVIEW

File: src/dispatchers/touchDispatcher.ts [hunk 1]
```diff
@@ -146,6 +146,9 @@ export module Dispatchers {
      * calls all the callbacks in the provided callbackSet.
      */
     private _measureAndDispatch(event: TouchEvent, callbackSet: Utils.CallbackSet<TouchCallback>) {
+      if (!this._translator.insideSVG(event)) {
```
Comment (jtlan) [inline src/dispatchers/touchDispatcher.ts]: Shouldn't this only apply on `touchstart`? See `Dispatcher.Mouse`'s logic when dealing with mouse events.

**Comment by jtlan** [comment]
Test(s)?

**Comment by crmorford** [comment]
Justin's safari fix got +1ed. After that merges into develop, can you merge develop into this, so I can see how they work together?

**Comment by crmorford** [comment]
also, gonna wait til tomorrow so sara from slate can look, since she filed the bug

**Comment by ztsai** [comment]
rebased to include Justin's fix

**Pull request merged**